### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by VanessaAoki**
# For this test branch, we implemented the following:

- [x] Add a todo.test.js file;
- [x] Add tests to add and remove items from the list;
- [x] Add tests to add and remove items from local storage; 
- [x] Set mock for localStorage and DOM manipulation;
- [x] Group list and localStorage tests into different describes.

## About not implemented navigation error
As you can see, all of our tests pass, but there are some errors displayed when running `$ npm test`.
After long research, we found that with the latest JSDOM update (to version 17.0.0) there is some sort of incompatibilities with Jest, as you can see in this [thread](https://github.com/jsdom/jsdom/issues/2524). 
So, for now, there is no fixing those issues, as the JSDOM/Jest maintainers did not yet fix the bug.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
+/*
```
Comment (rloterh) [inline jest.config.js:1]: - [x] Jest setup is throwing errors. Kindly ensure to fix the below issue:

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Nelsino & Vanessa, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [x] _Kindly check the comments under the review._


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: jest.config.js (hunk 1 above)
Comment (nelsinofrancisco) [reply jest.config.js:1]: Can you see the about not implemented navigator error in PR description, please? The tests are working but we have a bug when we try to use new JSDOM() to change the globals for window and document... we think it's a bug with the jest package. But you could give us some insight on how to solve this error too 😄

**Review by carlos-ssh — APPROVED** [review]
Hi @VanessaAoki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: jest.config.js (hunk 1 above)
Comment (rloterh) [reply jest.config.js:1]: Hello Team,

I've checked the this issue carefully and I realized there was a mistake in my review.
I sincerely apologize for this review mistake. Kindly note that this was not done deliberately
Cheers, and happy coding!

**Pull request merged**